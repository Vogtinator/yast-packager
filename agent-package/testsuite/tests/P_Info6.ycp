{
	return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"de","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
	return SCR::Read(.selGroups);
}
{
	return SCR::Execute(.selectSelInstall, "Standard", true );
}
{
	return SCR::Read(.selInstallSet);
}
{
	return SCR::Read(.selDependencies);
}
{
	return SCR::Execute(.selectSelInstall, "Minimal", false );
}
{
	return SCR::Read(.selInstallSet);
}
{
	return SCR::Read(.selDependencies);
}
{
	return SCR::Execute(.deselectSelInstall, "Minimal" );
}
{
	return SCR::Read(.selDependencies);
}
{
	return SCR::Execute(.selectSelInstall, "Kde", false );
}
{
	return SCR::Read(.selDependencies);
}
{
	return SCR::Read(.selInstallSet);
}
{
	return SCR::Execute(.deselectSelInstall, "Games" );
}
{
	return SCR::Read(.selDependencies);
}
{
	return SCR::Read(.selInstallSet);
}
{
	return SCR::Read(.selDelDesc, "default" );
}
{
	return SCR::Read(.selNotifyDesc, "default" );
}
