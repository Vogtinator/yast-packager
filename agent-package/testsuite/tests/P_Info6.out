(true)
([["Devel", " Entwicklung", "addon"], ["Games", " Spiele", "addon"], ["Gnome", " Gnome System", "addon"], ["Kde", " KDE komplett", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minimal System", "baseconf"], ["Minimal+X11", " Minimales graphisches System (ohne KDE) ", "baseconf"], ["Multimed", " Multimedia", "addon"], ["Multimedia", " Multimedia", "addon"], ["Network", " Netzwerk/Server", "addon"], ["Standard", " Standard-System", "baseconf"], ["default", " Standard-System mit Office", "baseconf"], ["xxxx", "  german xxxx", "baseconf"], ["zzz_Alles", " Alle Pakete (sehr zeitaufwendig)", "baseconf"]])
(true)
(["Kde-Desktop", "Standard"])
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(true)
(["Kde-Desktop", "Minimal", "Standard"])
($["ADD":[], "CONFLICT":[$["name":"Minimal", "selections":["Standard"]], $["name":"Standard", "selections":["Minimal"]]], "REQUIRE":[]])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["Kde", "Kde-Desktop", "Standard"])
(true)
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["Kde", "Kde-Desktop", "Standard"])
("")
("")
